homework/css/components/header.css

```diff
@@ -0,0 +1,175 @@
+header {
+  width: 100%;
+  background-color: rgba(67, 71, 67, 0.7);
+  grid-area: header;
+  display: grid;
+  position: relative;
+}
+
+.active {
+  background-color: #f9cb12;
+}
+
+header .logo {
+  grid-area: logo;
+} 
+
+header .logo .logo-image {
+  min-width: 90px;
+  height: 100%;
+  background-image: url("../../images/flight.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+header nav {
+  grid-area: nav;
+  display: flex;
+  justify-content: center;
+}
+
+header nav ul {
+  display: flex;
+  align-items: center;
+}
+
+header nav ul li {
+  cursor: pointer;
+  height: 100%;
+}
+
+ header nav ul li span {
+ display: inline-block;
+ line-height: 100px;
+ height: 100%;
+ padding: 0 20px;
+ transition: background-color 300ms ease-in-out;
+}
+
+
+@media (max-width: 575px) {
+  header {
+    grid-template-rows: 100px 50px;
+    grid-template-columns: 1fr; 
+    grid-template-areas: 
+      'logo'
+      'nav';
+  }
+
+  header .logo .slogan {
+    display: none;
+  }
+
+  header nav ul li span {
+    display: inline-block;
+    line-height: 50px;
+    height: 100%;
+    padding: 0 20px;
+  }
+
+  header .headerLogIn {
+    display: none;
+  }
+
+  .active {
+    background-color: #f9cb12;
+  }
+}
+
+@media (min-width: 576px) {
+  header {
+    grid-template-rows: 100px;
+    grid-template-columns: 1fr auto 1fr; 
+    grid-template-areas: 
+      'logo nav login';
+  }
+
+  header .logo .slogan {
+    display: none;
+  }
+
+  header .headerLogIn {
+    grid-area: login;
+    line-height: 100px;
+    position: absolute;
```

Comment:
positioning it absolute is not the most flexible way. You can replace all the .headerLogIn element styles except grid-area with 
`    align-self: center;
    justify-self: center;`

Reply:
Done